deploy/crds/saas.3scale.net_backends_crd.yaml

```diff
@@ -0,0 +1,270 @@
+                      type: string
+                      description: Label value used by grafana-operator for dashboard discovery
+            config:
+              type: object
+              properties:
+                rackEnv:
+                  type: string
+                  description: Rack environment (used for example for error-monitoring ID)
+                masterServiceId:
+                  type: integer
+                  description: Master service account ID in Porta
+                oauthMaxTokenSize:
+                  type: integer
+                  description: Oauth Max token size (bytes)
+                legacyReferrerFilters:
+                  type: boolean
+                  description: Enable (true) or disable (false) Legacy Referrer Filters
+            secret:
+              type: object
+              required:
+              - redisVaultPath
+              - systemEventsHookVaultPath
+              - internalApiVaultPath
+              properties:
+                redisVaultPath:
+                  type: string
+                  description: Vault Path with backend-redis secret definition
+                systemEventsHookVaultPath:
+                  type: string
+                  description: Vault Path with backend-system-events-hook secret definition
+                internalApiVaultPath:
+                  type: string
+                  description: Vault Path with backend-internal-api secret definition
+                errorMonitoringVaultPath:
+                  type: string
+                  description: Vault Path with backend-error-monitoring secret definition
+            errorMonitoringEnabled:
+              type: boolean
+              description: Mount (true) or not (false) backend-error-monitoring Secret on deployments
+            listener:
+              type: object
+              properties:
+                routeHost:
+                  type: string
+                  description: Host to configure on backend listener Route
+                replicas:
+                  type: integer
+                  description: Number of replicas
+                env:
+                  type: object
+                  description: Environment variables
+                  properties:
+                    logFormat:
+                      type: string
+                      description: Log format (text/json)
+                    listenerWorkers:
+                      type: integer
+                      description: Number of worker processes per listener pod
+                    redisAsync:
+                      type: boolean
+                      description: Enable (true) or disable (false) redis async mode
+                resources:
+                  type: object
+                  properties:
+                    requests:
+                      type: object
+                      properties:
+                        cpu:
+                          description: Override CPU requests
```

Comment:
As discussed over slack, I think it would best to use a string with the resource pattern.

https://github.com/3scale/saas-operator/commit/3dc0aa08d6ae4a29e5bc261ef7910c6d4e0e8d8c#diff-f07b2f94bdd30a5fa8627e8b2668143eR144

Reply:
Fixed https://github.com/3scale/saas-operator/pull/6/commits/d3c0c4d94ed9cdcb8362212919ba02ab36084c82